lib/const.rb

```diff
@@ -55,7 +55,7 @@
 CREW_PREFIX = ENV.fetch('CREW_PREFIX', '/usr/local')
 
 if CREW_PREFIX == '/usr/local'
-  CREW_BUILD_FROM_SOURCE = !!ENV['CREW_BUILD_FROM_SOURCE']&.eql?('1')
+  CREW_BUILD_FROM_SOURCE = !ENV['CREW_BUILD_FROM_SOURCE']&.eql?('1').nil?
```

Comment:
No easier way to do this?  Seems like overly complicated logic.

Reply:
I don't know? These were the automated rubocop replacements using our current rubocop linter config, so in theory this should have been triggered by the changes in any case...

Reply:
It appears we are mixing and matching methods and not being consistent.  I see nothing wrong with using `ENV.fetch` throughout.  Would this work?
```
 ENV.fetch('CREW_CACHE_BUILD', nil)&.eql?('1')
```

Reply:
We could try it!

Reply:
@supechicken what do you think?

Reply:
Like my view towards `Dir[]` and `Dir.glob()`, I prefer mixing them. (but I am okay with using one of them only :) )

Reply:
The DRY method does not apply here. :)

Reply:
As mentioned in the other comment, I've used this format as much as possible:

```ruby
CREW_VARIABLE = ENV.fetch('CREW_VARIABLE', '0').eql?('1')
```